Specs/Laser.md

```diff
@@ -0,0 +1,103 @@
+| 0:HasProperty | Variable | 0:DefaultInstanceBrowseName | 0:QualifiedName | 0:PropertyType |     |
+| 0:HasInterface | ObjectType | IMachineVendorNameplateType | | | |
+| 0:HasInterface | ObjectType | IMachineTagNameplateType | | | |
+| Applied from IMachineVendorNameplateType |     |     |     |     |     |
+| 0:HasProperty | Variable | 2:ProductInstanceUri | 0:String | 0:PropertyType | M, RO |
+| 0:HasProperty | Variable | 2:Manufacturer | 0:LocalizedText | 0:PropertyType | M, RO |
+| 0:HasProperty | Variable | 2:ManufacturerUri | 0:String | 0:PropertyType | O, RO |
+| 0:HasProperty | Variable | **2:Model** | 0:LocalizedText | 0:PropertyType | **M**, RO |
+| 0:HasProperty | Variable | **2:ProductCode** | 0:String | 0:PropertyType | **M**, RO |
+| 0:HasProperty | Variable | 2:HardwareRevision | 0:String | 0:PropertyType | O, RO |
+| 0:HasProperty | Variable | **2:SoftwareRevision** | 0:String | 0:PropertyType | **M**, RO |
+| 0:HasProperty | Variable | 2:DeviceClass | 0:String | 0:PropertyType | O, RO |
+| 0:HasProperty | Variable | 2:SerialNumber | 0:String | 0:PropertyType | M, RO |
+| 0:HasProperty | Variable | **YearOfConstruction** | UInt16 | 0:PropertyType | **M**, RO |
+| 0:HasProperty | Variable | MonthOfConstruction | Byte | 0:PropertyType | O, RO |
+| 0:HasProperty | Variable | InitialOperationDate | DateTime | 0:PropertyType | O, RO |
+| Applied from IMachineTagNameplateType |     |     |     |     |     |
+| 0:HasProperty | Variable | 2:AssetId | 0:String | 0:PropertyType | O, RW |
+| 0:HasProperty | Variable | 2:ComponentName | 0:LocalizedText | 0:PropertyType | O, RW |
+| 0:HasProperty | Variable | **Location** | 0:String | 0:PropertyType | **M**, RW |
+
+#### Special requirements for correct location of the machine icon on the dashboard
+
+The instance _Location_ property is evaluated according to the special requirements detailed [here](../Dashboard.md#location-of-fair-machine-and-software-icons-on-the-dashboard) to place a map icon for the machine.
+
+### Adaption of the provided information models for your purpose
+
+- The provided NodeSet contains the address space with the ObjectTypes, VariableTypes and DataTypes of the MachineTools companion specification and **must not** be changed.
+- Change the URI of the instance namespace (optional, but recommended)
+
+  `http://www.<MANUFACTURERDOMAIN>/example`
+
+- The rules for creating a URI can be found online. Please follow [this link](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier). In the case of a URL as a URI, it is not necessary that the site actually exists, it only identifies the machine uniquely. In order to simplify debugging it is recommended to choose a URI which allows an identification of the machine also for third persons, e.g. by including the company name.
+
+- The machine will get a unique NamespaceURI after being integrated to the datahub by addition of a suffix to ensure unique namespaces.
```

Comment:
... unique identifier like `clientId%3Dvdw%7Csmplsrv1%3Bnsu%3Dhttp%3A_2F_2Fwww.wenzel-group.com_2FWenzel_20LH_2087_2F%3Bi%3D66382` `<clientId><mqttusername><NamespaceUri><EntryNodeId>`
Whole sentence should be revised, as this is a leftover from the old datahub concept.